lib/brewdler/utils.rb

```diff
@@ -0,0 +1,10 @@
+module Brewdler
+  def self.system cmd, *args
+    pid = fork do
+      $stdout.reopen("/dev/null")
+      exec(cmd, *args)
+    end
```

Comment:
I'd probably still recommend `IO.popen` as it gives us more flexibility in future to do things like e.g. only print `stdout` on failure or a `--verbose`, combine `stderr` into the output etc. Your call though as you're doing the work :grinning: